examples/timeline/other/onTimeout.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE HTML>
```

Comment:
The demo as is appears to be taking an incredibly long time; is it possible to scale it down so that it takes less time to complete?

Also, that dialog 'Too many items loaded!' keeps on popping up. Very irritating if it's a long-running tests. If you can get the run-time down, then it's okay if it pops up 2-3 times.